packages/terra-form/tests/jest/components/Input.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Input from '../../../src/components/Input';
+
+it('should render a default component', () => {
+  const input = (<Input value="" />);
+  const wrapper = shallow(input);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render a Input with the rest of the props', () => {
+  const input = (
+    <Input
+      required
+      name="foo"
+      value="bar"
+    />);
+
+  const wrapper = shallow(input);
+  expect(wrapper).toMatchSnapshot();
+});
```

Comment:
Is there value in adding snapshot tests for the differences between controlled and uncontrolled components?